sdk/src/main/java/com/google/cloud/dataflow/sdk/util/common/Counter.java

```diff
@@ -129,6 +139,13 @@
    * @throws IllegalArgumentException if the aggregation kind is not supported
    */
   public static Counter<Long> longs(String name, AggregationKind kind) {
+    return new LongCounter(Name.createWithoutStructure(name), kind);
+  }
+
+  /**
+   * Like the above, only takes a {@link Counter.Name} instead of a string.
```

Comment:
Nit: `Like {@link #longs(String, AggregationKind)} ...` rather than "Like the above"